docs/src/dev/debugging.md

```diff
@@ -88,9 +88,8 @@ ERROR - ************************************
 This means the debugger is waiting for a connection. If you do not see this hang,
 then confirm that the debugger is enabled and installed prior to calling the core.
 
-You can also enable the debugger at runtime using the `Configure` routine, but use caution.
-Runtime enablement can skip the initial breakpoint and may cause security issues. For development,
-prefer force enablement.
+You can also enable the debugger at runtime using the `enable` routine, but use caution.
+Dynamic enablement should be carefully thought through to ensure proper platform security.
```

Comment:
Can you please end this so it's less open ended? Either point to an existing location talking about source debugger security considerations or briefly mention them here.

Reply:
Done, added a section adding a bit more details.